requirements/loggers.txt

```diff
@@ -1,2 +1,5 @@
 # test_tube>=0.7.5
-# trains>=0.14.1
\ No newline at end of file
+# trains>=0.14.1
+matplotlib
+wandb
```

Comment:
where is this used?

Reply:
`grep wandb -rn pl_bolts`
```
pl_bolts/callbacks/data_monitor.py:14:    import wandb
pl_bolts/callbacks/data_monitor.py:16:    wandb = None
pl_bolts/callbacks/data_monitor.py:91:                data={name: wandb.Histogram(tensor)}, commit=False,
```

Reply:
ok, is wandb the only one supporting histograms? I think that it shall be logger agnostic...
anyway, lest move it among logger? 
https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/requirements/loggers.txt

Reply:
so, shall we add `wandb` to requirements for now?